target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6018-fap650.dts

```diff
@@ -0,0 +1,458 @@
+// SPDX-License-Identifier: MIT, GPL-2.0 or later
+/* Copyright (c) 2023, Ruslan Isaev <legale.legale@gmail.com> */
+
+/dts-v1/;
+
+
+#include "ipq6018.dtsi"
+#include "ipq6018-cp-cpu.dtsi"
+#include "ipq6018-ess.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Yuncore FAP650";
+	compatible = "yuncore,fap650", "qcom,ipq6018-cp03", "qcom,ipq6018";
```

Comment:
Do you really need `"qcom,ipq6018-cp03"`

Reply:
this is cp03 device, i'm not sure if something use this.

Reply:
Its not, it was just loosely based off the CP03 reference board, drop it.

Reply:
ok. dropped.